0_main.R

```diff
@@ -61,11 +61,14 @@ plan(multisession, workers = 3)
 ## work on what the best integration between the development stratigies is 
```

Comment:
On line 52 above, is this setting up for parallel processing in future code versions? Currently the code as is is not set up for parallel processing from what I see... If that was a goal, one thing you could consider using is `furrr` functions in place of `purrr` functions, however I've noticed bugs recently when running functions using terra inside `furrr` functions so might need a different alternative.